frontend/public/components/operator-hub/operator-hub-subscribe.tsx

```diff
@@ -50,6 +50,21 @@ export const OperatorHubSubscribeForm = withFormState((props: OperatorHubSubscri
   }
 
   const {provider, channels = [], packageName} = props.packageManifest.data.status;
+  const srcProvider = _.get(props.packageManifest, 'metadata.labels.opsrc-provider');
+  let providerType;
+    switch (srcProvider) {
+      case 'redhat':
+        providerType = 'Red Hat';
+        break;
+      case 'certified':
+        providerType = 'Certified';
+        break;
+      case 'community':
+        providerType = 'Community';
+        break;
+      default:
+        providerType = 'Custom';
+    }
```

Comment:
we should make this a util since it is used here and in the operator-hub-page.

Reply:
Agreed! Do you want me to wait for your stuff to merge first or can you include that in your PR?

Reply:
Added the util to https://github.com/openshift/console/pull/1051